java/code/src/com/suse/manager/webui/websocket/VirtNotifications.java

```diff
@@ -139,8 +139,7 @@ public void onMessage(Session session, String messageBody) {
                     // Send out a list of the latest pending actions to display
                     Server server = SystemManager.lookupByIdAndUser(newId, userOpt.get());
                     List<ServerAction> serverActions = ActionFactory.listServerActionsForServer(server,
-                            Arrays.asList(ActionFactory.STATUS_QUEUED,
-                                          ActionFactory.STATUS_PICKEDUP,
+                            Arrays.asList(ActionFactory.STATUS_QUEUED, ActionFactory.STATUS_PICKED_UP,
                                           ActionFactory.STATUS_PICKED_UP));
```

Comment:
It seems we're duplicating `ActionFactory.STATUS_PICKED_UP` here.

IIUC this shoud be:
 ```suggestion
                             Arrays.asList(ActionFactory.STATUS_QUEUED,
                                           ActionFactory.STATUS_PICKED_UP));
```

Reply:
Hmmm, was this change missed after the force push?